app/2.0.x/pdk/kong.nginx.md

```diff
@@ -32,4 +38,5 @@ Returns the current Nginx subsystem this function is called from: "http"
 kong.nginx.get_subsystem() -- "http"
 ```
 
-[Back to top](#kong.nginx)
+[Back to top](#kongnginx)
```

Comment:
This link was pointing to a non-existing anchor (the new one is automatically generated from the page title)